src/components/general/IntroContent/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import './IntroContent.scss'
+import Text from '../Text'
+import BackgroundImage from '../../../assets/IntroBackground.png'
+
+interface IIntroContent {
+  icon: React.ReactNode
+  title: string
+  subtitle: string
+  button: React.ReactNode
+}
+
+const IntroContent: React.FC<IIntroContent> = ({ icon, title, subtitle, button }) => {
+  return (
+    <>
```

Comment:
I guess we don't need the fragment here as the div is wrapping the whole component.